project-p2-layouts-master/code/contact.css

```diff
@@ -0,0 +1,222 @@
+body {
+    background: #E6E8E9;
+    color: #000000;
+    font-family: 'Open Sans Condensed', sans-serif;
+    margin: 0;
+  }
+  
+  /*** Header ***/
+  
+  header {
+    display: flex;
+    background: #24292d;
+    height: 60px;
+  }
```

Comment:
Indentation - see code guidelines for specification

Reply:
Hi Mathilda, Thank you so much for the feedback! It helps a lot :) I'm slightly confused as to what I need to change here. I looked at the code guidelines but don't see the issue with the indentation? 

Could you be more specific please? Thank you :)

Reply:
Oh yes, sure! The closing tag should be on the same level of indentation as the opening line. If the code is not in media queries you don't have to indent the element.

Example 1. Here I took away the indentation from the closing tag
```
body {
    background: #E6E8E9;
    color: #000000;
    font-family: 'Open Sans Condensed', sans-serif;
    margin: 0;
}
```

Example 2. Here I took away the indentation from the whole element.
```
header {
    display: flex;
    background: #24292d;
    height: 60px;
}
```

If you were to have a media query, you should indent your element like this:

```
@media only screen and (max-width: 600px) {
    body {
        background-color: lightblue;
    }
}
```

Reply:
This makes a lot more sense! I see it now :) Thank you very much!!